README.md

```diff
@@ -1,77 +1,42 @@
-[![Build Status](https://travis-ci.org/cloudinary/cloudinary_js.svg)](https://travis-ci.org/cloudinary/cloudinary_js) [![npm](https://img.shields.io/npm/v/cloudinary_js.svg?maxAge=2592000)]() [![license](https://img.shields.io/github/license/cloudinary/cloudinary_js.svg?maxAge=2592000)]()
+Cloudinary Javascript SDKs (Legacy)
+==========================
 
-# Cloudinary Client Side JavaScript Library
+#Note About Legacy SDKs
```

Comment:
Should this be a # tag? 
because right now it appears as `#Note about`

should it be like this?

# Note about...